src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck machine1 = new Truck();
+        machine1.setName("Мишаня");
+        Excavator machine2 = new Excavator();
+        machine2.setName("Ванёк");
+        Bulldozer machine3 = new Bulldozer();
```

Comment:
let's see checklist (common mistakes doc) - are all points from it implemented in the solution?